cmd/zrepl/mgmt_conn.c

```diff
@@ -557,6 +558,27 @@ uzfs_zvol_rebuild_status(uzfs_mgmt_conn_t *conn, zvol_io_hdr_t *hdrp,
 	return (reply_data(conn, &hdr, &status_ack, sizeof (status_ack)));
 }
 
+static int
+uzfs_zvol_stats(uzfs_mgmt_conn_t *conn, zvol_io_hdr_t *hdrp,
+    const char *name, zvol_info_t *zinfo)
+{
+	zvol_io_hdr_t	hdr;
+	zvol_op_stat_t	stat;
+
+	strncpy(stat.label, "used", sizeof (stat.label));
+	stat.value = dsl_dir_phys(
+	    zinfo->zv->zv_objset->os_dsl_dataset->ds_dir)->dd_used_bytes;
+
+	bzero(&hdr, sizeof (hdr));
+	hdr.version = REPLICA_VERSION;
+	hdr.opcode = hdrp->opcode;
+	hdr.io_seq = hdrp->io_seq;
+	hdr.len = sizeof (zvol_op_stat_t);
+	hdr.status = ZVOL_OP_STATUS_OK;
+
+	return (reply_data(conn, &hdr, &stat, sizeof (stat)));
```

Comment:
I hope we are not expecting any error here as it is not handled up the line in caller sequence.